Formula/s6.rb

```diff
@@ -0,0 +1,30 @@
+class S6 < Formula
+  desc "Small & secure supervision software suite."
+  homepage "http://skarnet.org/software/s6/"
+  url "http://skarnet.org/software/s6/s6-2.3.0.0.tar.gz"
+  sha256 "49c9a0476d93c6279c776ca5ae9c49b3fd39c356eda674828aebf78f8488e8e6"
+  head "git://git.skarnet.org/s6"
+
+  depends_on "skalibs"
+  depends_on "execline"
+
+  def install
+    system "./configure",
+      "--prefix=#{prefix}",
+      "--with-sysdeps=#{Formula["skalibs"].opt_prefix}/lib/skalibs/sysdeps",
+      "--with-lib=#{Formula["skalibs"].opt_prefix}/lib",
+      "--with-lib=#{Formula["execline"].opt_prefix}/lib/execline",
+      "--disable-shared"
+    system "make", "install"
+
+    # Some S6 tools expect execline binaries to be on the path
+    bin.env_script_all_files(libexec/"bin", :PATH => "#{Formula["execline"].libexec}:$PATH")
+    sbin.env_script_all_files(libexec/"sbin", :PATH => "#{Formula["execline"].libexec}:$PATH")
+  end
+
+  test do
```

Comment:
Thanks, the return status check was indeed redundant; it has to be `pipe_output`, though, the command takes input until the pipe closes.